src/main/resources/application.yml

```diff
@@ -13,9 +13,8 @@ env:
       pass: ${DATABASE_USER_PASSWORD:wmpwd}
       user: ${DATABASE_USER:wmuser}
   tracing:
-    enabled: ${CLOUD_TRACE_ENABLED:false}
-    probability: ${SAMPLING_PROBABILITY:0}
-    project-id: ${SERVICE_GOOGLE_PROJECT:broad-dsde-dev}
```

Comment:
out of scope, but it looks like this is the only place we're using this var so we can probably clean it out of helm + helmfile. can you ticket that so it doesn't get lost?

Reply:
I'll do it as a part of this ticket since I'm deprecating it.
Do I need to wait some amount of time after this is merged before updating the helm chart, or will any use of the new helm chart also pick up the latest WM binary?

Reply:
You'll have to wait until this version of the helm chart is rolled out everywhere before removing the helmfile value, you can see the current state of things in the [helmfile/versions dir](https://github.com/broadinstitute/terra-helmfile/tree/master/versions). I think it's usually ~1 week for releases to work their way to "prod"
The binary should be rolled out together with the chart though